cmd/piper.go

```diff
@@ -154,6 +155,7 @@ func PrepareConfig(cmd *cobra.Command, metadata *config.StepData, stepName strin
 		}
 	}
 
+	stepConfig.Config = convertTypes(stepConfig.Config, options)
```

Comment:
Maybe we can improve the naming here.  E.g. options is quite general.  It took me a while to figure out what contains the values and where to find the types...

Reply:
Yes. It also doesn't help that "options" and "config" is synonymous, at least as far as I am concerned. But do you have a suggestion for better naming for the code that this PR adds? Like perhaps convertTypes -> convertMapEntriesToOptionsFieldTypes() or something?